ops/cli.py

```diff
@@ -0,0 +1,249 @@
+"""WFL Deployment Script 
```

Comment:
To facilitate ease of use, perhaps include a python3 shebang so that we can just call it:
```
./ops/cli.py --help
```

Reply:
Thanks! Updated :)